acme_diags/driver/streamflow_driver.py

```diff
@@ -99,17 +124,20 @@ def run_diag(parameter):
             # the data has been reorganized to a 1380 * 30961 matrix. 1380 is the month
             # number from 1901.1 to 2015.12. 30961 include two columns for year and month plus
             # streamflow at 30959 gauge locations reported by GSIM
-            ref_mat = scipy.io.loadmat(parameter.ref_mat_file)
+            ref_mat_file = '/{}/GSIM/GSIM_198601_199512.mat'.format(parameter.reference_data_path.strip('/'))
+            parameter.ref_name_yrs = 'GSIM ({}-{})'.format(parameter.ref_start_yr, parameter.ref_end_yr)
```

Comment:
The ref name is set to be `GSIM`. Should I change this to be superseded by `parameter.short_ref_name`?